solana/src/main/java/com/solana/core/Message.kt

```diff
@@ -111,17 +116,17 @@ class Message {
         return out.array()
     }
 
-    fun setFeePayer(feePayer: Account?) {
-        this.feePayer = feePayer
-    }
-
     private fun getAccountKeys(): List<AccountMeta> {
         val keysList: MutableList<AccountMeta> = accountKeys.list
-        val feePayerIndex = findAccountIndex(keysList, feePayer!!.publicKey)
         val newList: MutableList<AccountMeta> = ArrayList()
-        val feePayerMeta = keysList[feePayerIndex]
-        newList.add(AccountMeta(feePayerMeta.publicKey, true, true))
-        keysList.removeAt(feePayerIndex)
+        try {
+            val feePayerIndex = findAccountIndex(keysList, feePayer!!)
+            val feePayerMeta = keysList[feePayerIndex]
+            newList.add(AccountMeta(feePayerMeta.publicKey, true, true))
+            keysList.removeAt(feePayerIndex)
+        } catch(e: RuntimeException) { // Fee payer not yet in list
```

Comment:
I think it's better if `findAccountIndex` returns -1 when it doesn't find the account instead of throwing. Then this would be an if else and not a try catch. Curious what maintainers think about that change.